scripts/install/komga.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+. /etc/swizzin/sources/globals.sh
+. /etc/swizzin/sources/functions/utils
```

Comment:
Not sure if this is here for the user functions, but those are now in globals too